src/components/Badge.vue

```diff
@@ -53,20 +54,18 @@ export default {
 
   @include font-styles(badge);
   display: inline-block;
-  padding: 2px 10px;
+  padding: 2px 4px;
   white-space: nowrap;
-  background: none;
-  border-radius: var(--badge-border-radius, calc(#{$border-radius} - 1px));
-  border-style: var(--badge-border-style, solid);
-  border-width: var(--badge-border-width, 1px);
-  margin-left: 10px;
-  color: var(--badge-color);
-
-  .theme-dark & {
-    --badge-color: var(--badge-dark-color);
+  border-radius: var(--badge-border-radius, $border-radius);
+  border: none;
+  margin: auto;
+  margin-left: 5px;
+  color: var(--colors-button-text, var(--color-button-text));
+  background-color: var(--badge-color);
```

Comment:
Now that we're using the badge color as a background fill instead of a border, we'll likely need to re-think the color values themselves, since I don't think the contrast ratio will be high enough in terms of accessibility.